website/content/commands/server-snapshot.mdx

```diff
@@ -15,7 +15,12 @@ Write a backup of the server data
 
 ## Usage
 
-Usage: `waypoint server snapshot [options]`
+Usage: `waypoint server snapshot [<filename>]`
+
+    Generate a snapshot from the current server and write it to a file specified
+    by the given name. If no name is specified and standard out is not a terminal,
+    the backup will be written to standard out. Using a name of '-' will force writing
+    to standard out.
```

Comment:
Indentation causes codeblock 
<img width="1050" alt="Screen Shot 2021-08-10 at 9 49 50 AM" src="https://user-images.githubusercontent.com/8404559/128878822-38adfe21-273e-41de-a838-23f7af7ad824.png">

Reply:
Hey! This is because the formatting in our actual help text is off.
![image](https://user-images.githubusercontent.com/8461333/128888053-ad3a10c7-2c37-4be9-b1e8-29877a9647c2.png)

If I fix the spacing in the command from a tab to a space, then it fixes both things.

Reply:
nice! this is revealing a lot of stuff that should be fixed with our docs it looks like 😄 I can fix this all up